### PR TITLE
V9.3.2 add config 'fields' to vk provider

### PR DESCRIPTION
**Pull request by slaawwa**
Update response data from vk (move 'uid' to 'id')
And add custom config 'fields'

### CODE REVIEW

File: lib/providers/vk.js [hunk 1]
```diff
@@ -26,11 +26,15 @@ exports = module.exports = function (options) {
...
+            if (options.fields) {
+                query.fields = options.fields;
+            }
+
             const data = await get('https://api.vk.com/method/users.get', query);
             const profile = data.response[0];
 
             credentials.profile = {
-                id: profile.uid,
+                id: profile.id,
```
Comment (AdriVanHoudt) [inline lib/providers/vk.js]: Afaik the api returns uid not id?
https://vk.com/dev/fields

Comment (slaawwa) [reply lib/providers/vk.js]: Page move from
https://vk.com/dev/fields
to
https://vk.com/dev/objects/user

```
https://vk.com/dev/users.get

Returns a list of user objects. 

A deactivated field may be returned with the value deleted or banned if a user has been suspended.
```
I think vk team move field  **uid** to field **id** in last version

Comment (AdriVanHoudt) [reply lib/providers/vk.js]: I can get to both pages by clicking through the docs so confused haha
Do you use the api? Can you validate that this change works?

Comment (slaawwa) [reply lib/providers/vk.js]: @AdriVanHoudt, yes. I use it) And this is reason why I create this pull-request)

Comment (slaawwa) [reply lib/providers/vk.js]: @AdriVanHoudt, maybe try get `id` field and if it's field not exist try get `uid` field?

Comment (AdriVanHoudt) [reply lib/providers/vk.js]: Sounds like a plan, if you change that I will merge 🎉

Comment (slaawwa) [reply lib/providers/vk.js]: @AdriVanHoudt, what the problem with travis tests?

**Pull request merged**